nginx/static/batch.html

```diff
@@ -0,0 +1,151 @@
+
+
+<div class="main text-center">
+  <div class="home text-center">
+      <div class="container">
+          <div class="row">
+              <div class="col-md-12">
+                  <h1 class="headline">Batch Search Engine</h1>
+              </div>
+          </div>
+
+          <div class="row">
+              <div class="col-md-12">
+                  <form action="" class="BFEX-search-form">
+                      <input type="search" placeholder="Search BFEX..." class="BFEX-search-input" id="BFEX-search-input">
+                      <button class="button btn btn-success btn-BFEX">BFEX Search</button>
+                  </form>
+              </div>
+          </div>
+
+          <div class="row">
+              <div class="col-md-12">
+                  <footer class="attribution">
+                      <p>Designed and coded by TEAM BFEX</p>
+                  </footer>
+              </div>
+          </div>
+      </div>
+  </div>
+
+<!-- Results Page -->
+
+  <div class="result hidden">
+      <header class="header">
+          <div class="container">
+              <div class="row">
+                  <div class="col-md-12">
+                      <form action="" class="result-BFEX-search-form">
+                          <input type="text" placeholder="Search BFEX..." class="result-BFEX-search-form-input" id="result-BFEX-search-form-input">
+                          <button class="btn btn-success result-btn-BFEX">BFEX Search</button>
+                      </form>
+                  </div>
+                  <div class="col-md-12">
+                      <h1>BFEX Search</h1>
+                  </div>		
+              </div>
+          </div>
+      </header>	
+      
+      <div class="container">
+          <section class="search-results">
+              <div class="one-half column">
+                  <div class="display-results"></div>
+                  <table id="table">
+                    <thead>
+                    <tr>
+                        <th data-field="name">Name</th>
+                        <th data-field="keywords">approach</th>
+                    </tr>
+                    </thead>
+                </table>
+              </div>
+          </section>
+      
+      </div> <!-- Container -->	
+  </div> <!-- Results -->
+</div>
+<link rel="stylesheet" href="https://rawgit.com/wenzhixin/bootstrap-table/master/src/bootstrap-table.css" />
+<script src="https://rawgit.com/wenzhixin/bootstrap-table/master/src/bootstrap-table.js"></script>
+
+<script type="text/javascript">
+
+window.onload = function() {
+    document.getElementById("BFEX-search-input").focus();
+};
+$('#table').bootstrapTable();
+  function ajax(keyword){
+  var approaches = {0:'generic',1:'rake',2:'textrank',3:'tf-idf'}
+  return $.when(
+      $.ajax({url: "http://localhost:8901/search?query="+keyword,dataType:"json"}),
```

Comment:
This needs to be a variable or something... This only works when I'm running the server and ui on the same machine. When a user requests the page from our server, these will try to run requests on their local machine, and fail.